### ISSUE TITLE
Parent view bindings overwriting child view bindings

### ISSUE THREAD

**Issue by hotmeteor** [NYTimes/backbone.stickit#256]
I have a pretty common scenario... parent view contains many child views. Both the parent and child have models that share common attribute names like "name", "description", etc.

I'm using Stickit on both views, but I'm finding that the parent view's bindings are overwriting all of the child view's bindings. For example, if the parent is called "Category 1" then all of the children views have that name as well, instead of their actual name.

Basically, it seems like the bindings aren't scoping to their own views. Am I doing something incorrectly? Thanks!
